src/etherscan/mainnet_0x853844459106feefd8C7C4cC34066bFBC0531722/DefaultReserveInterestRateStrategy/lib/protocol-v2/contracts/dependencies/openzeppelin/contracts/IERC20.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: agpl-3.0
```

Comment:
What's the the Etherscan folder here with a bunch of files in it? What's it meant for?